Installer/MainWindow.xaml.cs

```diff
@@ -0,0 +1,257 @@
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.IO;
+using System.IO.Compression;
+using Path = System.IO.Path;
+using IWshRuntimeLibrary;
+using File = System.IO.File;
+
+namespace Installer
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        //Moving Window
+        private bool holdingDown;
+        private Point lm = new Point();
+
+        //Pages
+        public enum Page{ About, SelectFolder, Confirm, Extracting,Finished}
+        private Page currentPage;
+
+        //
+        private string vtFolder;
+        public MainWindow()
+        {
+            InitializeComponent();
+            SwitchPage();
+            /*
+           
+            */
+        }
+        private void Window_Initialized(object sender, EventArgs e)
+        {
+            
+        }
+
+        private string FindVTOL()
+        {
+            string regPath = (string)Registry.GetValue(
+                @"HKEY_LOCAL_MACHINE\SOFTWARE\Valve\Steam",
+                @"InstallPath",
+                @"NULL");
+
+            string gameFolder = File.ReadAllText(regPath + @"\steamapps\libraryfolders.vdf").Split('"')[13];
+            string[] split = gameFolder.Split('\\');
+            string result = "";
+            for (int i = 0; i < split.Length; i++)
+            {
+                if (!string.IsNullOrWhiteSpace(split[i]))
+                    result += split[i] + @"\";
+            }
+            result += @"steamapps\common\VTOL VR\";
+            return result;
+            
+        }
+        private void InstallFiles()
+        {
+            if (Directory.Exists(vtFolder + @"VTOLVR_ModLoader"))
+            {
+                //It must be already installed
+                SetProgress(100);
+                currentPage++;
+                SwitchPage();
+                return;
+            }
+            SetProgress(0);
+            //Extracting the zip from resources to files
+            string path = vtFolder + @"modloader.zip";
+            File.WriteAllBytes(path, Properties.Resources.ModLoader);
+            
+            //Stopping a possiable error
+            if (File.Exists(vtFolder + @"VTOLVR_Data\Plugins\discord-rpc.dll"))
+                File.Delete(vtFolder + @"VTOLVR_Data\Plugins\discord-rpc.dll");
+
+            ZipFile.ExtractToDirectory(path, vtFolder);
+            SetProgress(50);
+            File.Delete(path);
+            SetProgress(75);
+
+            if (dShortcut.IsChecked == true)
+                CreateShortcut(
+                    Environment.GetFolderPath(Environment.SpecialFolder.DesktopDirectory) + @"\VTOL VR Mod Loader.lnk",
+                    vtFolder + @"VTOLVR_ModLoader\VTOLVR-ModLoader.exe");
+            if (smShortcut.IsChecked == true)
```

Comment:
A comparison isn't needed here.. You could just use `dShortcut.IsChecked` instead of `dShortcut.IsChecked == true`

Reply:
`dShortcut.IsChecked == true` is needed because its a `bool?`.
So it can be true, false or null.